pandas/core/indexing.py

```diff
@@ -1920,11 +1920,35 @@ def _convert_key(self, key, is_setter=False):
 
 
 class _iAtIndexer(_ScalarAccessIndexer):
-    """Fast integer location scalar accessor.
+    """
+    Selects a single value for a row/column pair by integer position.
 
-    Similarly to ``iloc``, ``iat`` provides **integer** based lookups.
-    You can also set using these indexers.
+    Useful if performance is a major concern and you only need to
```

Comment:
I would be a bit more careful with "major concern", as if performance is really important, you typically should not use this indexer many times. But, it is certainly useful to mention that `iat` is faster than `iloc` when you know you want to access only a single scalar.